### PR TITLE
Update query-parameters.md

### PR DESCRIPTION
**Pull request by DanijelkMSFT**
extended the example of SkuID search with the needed parameters ConsistencyLevel: eventual in header and &$count=true in the query. As this is an advanced query and will fail when more results are found. Like described here https://docs.microsoft.com/en-us/graph/aad-advanced-queries

### CODE REVIEW

**Comment by Lauragra** [comment]
@FaithOmbongi , can you verify that this update looks okay? Thanks.

File: concepts/query-parameters.md [hunk 1]
```diff
@@ -176,8 +176,11 @@ GET https://graph.microsoft.com/v1.0/users?$filter=imAddresses/any(s:s eq 'admin
 
 The **assignedLicenses** property of the user resource contains a collection of **assignedLicense** objects, a complex type with two properties, **skuId** and **disabledPlans**. The following query retrieves only users with an assigned license identified by the **skuId** `184efa21-98c3-4e5d-95ab-d07053a96e67`.
 
+>**Note:** Be sure to add the `&$count=true` parameter and the `ConsistencyLevel: eventual` header. This is an advanced query and will fail if too many results are found.
```
Comment (Licantrop0) [inline concepts/query-parameters.md:179]:
```suggestion
>**Note:** Be sure to add the `&$count=true` parameter and the `ConsistencyLevel: eventual` header. This is an [advanced query](/graph/aad-advanced-queries) and will fail if too many results are found.
```

**Comment by Licantrop0** [comment]
I didn't know about this limitation, but it looks good to me (I added the link to the advanced queries page).
 /cc @kksrivatsa

**Review by FaithOmbongi — APPROVED** [review]
Thank you for flagging this and raising the fix @DanijelkMSFT. I've approved with suggestions.

I'll wait to hear from the PG if there's a quantity that is the ceiling for "too many results" and leave any further suggestions as applicable.

File: concepts/query-parameters.md [hunk 2]
```diff
@@ -176,8 +176,11 @@ GET https://graph.microsoft.com/v1.0/users?$filter=imAddresses/any(s:s eq 'admin
 
 The **assignedLicenses** property of the user resource contains a collection of **assignedLicense** objects, a complex type with two properties, **skuId** and **disabledPlans**. The following query retrieves only users with an assigned license identified by the **skuId** `184efa21-98c3-4e5d-95ab-d07053a96e67`.
 
+>**Note:** Be sure to add the `&$count=true` parameter and the `ConsistencyLevel: eventual` header. This is an [advanced query](/graph/aad-advanced-queries) and will fail if too many results are found.
+
 ```msgraph-interactive
-GET https://graph.microsoft.com/v1.0/users?$filter=assignedLicenses/any(s:s/skuId eq 184efa21-98c3-4e5d-95ab-d07053a96e67)
+GET https://graph.microsoft.com/v1.0/users?$filter=assignedLicenses/any(s:s/skuId eq 184efa21-98c3-4e5d-95ab-d07053a96e67))&$count=true
```
Comment (FaithOmbongi) [inline concepts/query-parameters.md:182]: Removed the extra bracket
```suggestion
GET https://graph.microsoft.com/v1.0/users?$filter=assignedLicenses/any(s:s/skuId eq 184efa21-98c3-4e5d-95ab-d07053a96e67)&$count=true
```

**Comment by FaithOmbongi** [comment]
Added the **do not merge** and **awaiting internal feedback** labels to allow for internal discussion.

**Comment by FaithOmbongi** [comment]
Thank you @DanijelkMSFT for your effort to highlight this. We've confirmed that the behavior is a bug and will be fixed soon.  Closing this PR but we look forward to your future PRs :)

**Pull request closed without merging**